### ISSUE TITLE
Increase the movement speed of unit just a tiny bit

### ISSUE THREAD

**Issue by kazgaroth011** [SWY1985/CivOne#162]
It's almost perfect.

**Comment by SWY1985** [comment]
It is perfect on my main computer, but it slows down when testing on my tablet.
I still need to add a setting to skip frames in order to make the unit movement faster...

**Comment by kazgaroth011** [comment]
This was probably also an issue 25 years ago, but still I do remember even from back then that the movement was just a bit faster. Just deal with this when you sort out other issues, this has a very low priority.

**Comment by iStruggle** [comment]
I was just about to post to say I thought it was actually too fast, but I checked it again and it was just a smidge too slow as axx says.  Then, I switched back and forth between the civone window and it sped up again.

**Comment by SWY1985** [comment]
With Dosbox, it runs quick on one computer and slow on another. I'm not even sure if there is a fixed speed at which it should run.
CivOne tends to slow down sometimes though... I'm not sure if I can really fix that. All I can do is add an option to skip frames so it doesn't slow down too much on slow computers. I've got an old netbook that I started testing on and that system really needs a frameskip patch.
